### DISCUSSION TITLE
Kibana only user

### DISCUSSION THREAD

**Discussion by m9o7h** [Security-Onion-Solutions/securityonion#7653] [Unsupported Versions] [answered]
Can we create a user in kibana that can login without login in SOC first?

**Comment by dougburks** [comment, marked as answer]
Technically, you could create a Kibana-only user account. However, that Kibana-only user would still have to authenticate to SOC in order to get to Kibana. For more information, please see https://docs.securityonion.net/en/2.3/so-elastic-auth.html.

**Reply by m9o7h** [reply]
I've created an account in kibana UI then tried to login through SOC and i got credentials invalid

**Reply by dougburks** [reply]
Yes, that's what I was trying to say in my previous reply. Quoting from https://docs.securityonion.net/en/2.3/so-elastic-auth.html since it explains it more thoroughly:
> Please note that if Elastic auth is enabled and you add a new user directly in [Kibana](https://docs.securityonion.net/en/2.3/kibana.html#kibana) via the [Kibana](https://docs.securityonion.net/en/2.3/kibana.html#kibana) Users page, then that new user will only have access to Kibana and no other apps. If you want the user to have access to all apps, make sure you add the user as shown in the [Adding Accounts](https://docs.securityonion.net/en/2.3/adding-accounts.html#adding-accounts) section.
